packages/mwp-api-state/src/cache/util.test.js

```diff
@@ -34,8 +34,8 @@ describe('cache utils', () => {
 		const cache = makeCache();
 		const query = { foo: 'baz' };
 		const response = 'bar';
-		return cacheWriter(cache)(query, response)
-			.then(() => cache.get(JSON.stringify(query)))
+		return cacheWriter(cache, 'memberId')(query, response)
+			.then(() => cache.get(`memberId${JSON.stringify(query)}`))
```

Comment:
mainly because it's a test script so i wasn't sure that I wanted to hide the 'expected' string format... can argue for either option, but I figure it's probably ok to break this test when `makeKey` changes